components/org.wso2.carbon.identity.policy.password/src/main/java/org/wso2/carbon/identity/policy/password/history/util/DatabaseUtil.java

```diff
@@ -0,0 +1,68 @@
+package org.wso2.carbon.identity.policy.password.history.util;
+
+import org.apache.commons.logging.Log;
```

Comment:
Shouldn't this class be placed in a common place. This is not something specific to password history. More use cases will need this.